web/crux-ui/src/pages/deployments/[deploymentId].tsx

```diff
@@ -108,7 +108,7 @@ const DeploymentDetailsPage = (props: DeploymentDetailsPageProps) => {
       return
     }
 
-    const result = await startDeployment(router, handleApiError, deployment.id, state.deployInstances)
+    const result = await startDeployment(router, t, deployment.id, state.deployInstances)
```

Comment:
If this depends so much on the context we should move it into the react component, so we don't have to pass the router and the translator.